src/blocks/chain4u.rs

```diff
@@ -16,6 +16,7 @@ use crate::{
         sector::PoStProof,
     },
 };
+use chain4u::header::GENESIS_BLOCK_PARENTS;
```

Comment:
nit: guess this slipped through the cracks in the original PR, while I like the name, module name with digits looks kinda ugly. But concise :D